java/test/src/main/java/org/ray/api/test/DynamicResourceTest.java

```diff
@@ -31,15 +31,15 @@ public void testSetResource() {
 
     Ray.setResource("A", 10.0);
 
-    // Assert node info.
-    List<NodeInfo> nodes = Ray.getRuntimeContext().getAllNodeInfo();
-    Assert.assertEquals(nodes.size(), 1);
-    Assert.assertEquals(nodes.get(0).resources.get("A"), 10.0);
-
     // Assert ray call result.
     result = Ray.wait(ImmutableList.of(obj), 1, 1000);
     Assert.assertEquals(result.getReady().size(), 1);
     Assert.assertEquals(Ray.get(obj.getId()), "hi");
+
+    // Assert node info.
+    List<NodeInfo> nodes = Ray.getRuntimeContext().getAllNodeInfo();
```

Comment:
even if put here, we still need to sleep a while to wait for the resource to be released?

Reply:
It is total resource of the node now.

But it makes sense to add a sleep here because we'll assert available resource later.

Reply:
use `waitForCondition`, and move this back right after `setResource`?